.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["es2015", "react", "stage-1"]
```

Comment:
Yes, that was added for jest. And we definitely should use that for babel-loader in webpack config.